.eslintrc.js

```diff
@@ -64,7 +64,7 @@ module.exports = /** @type {Config} */ ({
     'plugin:eslint-plugin-import/recommended',
     'plugin:eslint-plugin-import/typescript',
     'eslint-config-airbnb',
-    './eslint/config-airbnb-typescript.js',
+    require.resolve('./eslint/config-airbnb-typescript.js'),
```

Comment:
This makes sure that the `.eslintrc.js` file can be imported from other repos without eslint trying to search for `config-airbnb-typescript.js` file in the local folders.

Reply:
in eslint 9 we will be able to just list the preset as an object